akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -66,11 +65,8 @@ trait FileAndResourceDirectives {
       if (file.isFile && file.canRead)
         conditionalFor(file.length, file.lastModified) {
           if (file.length > 0) {
-            withRangeSupportAndPrecompressedMediaTypeSupportAndExtractSettings { settings ⇒
-              complete {
-                HttpEntity.Default(contentType, file.length,
-                  FileIO.fromPath(file.toPath).withAttributes(ActorAttributes.dispatcher(settings.fileIODispatcher)))
```

Comment:
remove the dispatcher attribute rendering the `file-io-dispatcher` setting effectively unused